src/main/java/de/dataelementhub/rest/controller/v1/ExportController.java

```diff
@@ -0,0 +1,96 @@
+package de.dataelementhub.rest.controller.v1;
+
+import de.dataelementhub.model.dto.export.ExportInfo;
+import de.dataelementhub.model.dto.export.ExportRequest;
+import de.dataelementhub.model.service.ExportService;
+import de.dataelementhub.rest.DataElementHubRestApplication;
+import java.sql.Timestamp;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.security.SecurityProperties;
+import org.springframework.core.annotation.Order;
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/v1")
+public class ExportController {
+
+  private final ExportService exportService;
+
+  @Autowired
+  public ExportController(ExportService exportService) {
+    this.exportService = exportService;
+  }
+
+  public static String exportDirectory = System.getProperty("user.dir") + "/uploads/export/";
+
+  /**
+   * Return an overview of all exports.
+   */
+  @Order(SecurityProperties.BASIC_AUTH_ORDER)
+  @GetMapping(value = "/export")
+  public ResponseEntity<List<ExportInfo>> allExports() {
+    Integer userId = DataElementHubRestApplication.getCurrentUser().getId();
+    List<ExportInfo> exportDescriptions = exportService.allExports(userId, exportDirectory);
+    return new ResponseEntity<List<ExportInfo>>(exportDescriptions, HttpStatus.ACCEPTED);
+  }
+
+  /**
+   * Start an export process for all given elements.
+   */
+  @Order(SecurityProperties.BASIC_AUTH_ORDER)
+  @PostMapping(value = "/export")
+  public ResponseEntity<String> export(@RequestBody ExportRequest exportRequest,
+      @RequestParam String format, @RequestParam Boolean fullExport) {
+    int numberOfNamespacesExportedFrom = exportRequest.getElementUrns().stream()
+        .map(e -> e.split(":")[1]).collect(Collectors.toSet()).size();
+    if (numberOfNamespacesExportedFrom > 1) {
+      return new ResponseEntity<>("Export from more than one namespace is forbidden",
+          HttpStatus.NOT_ACCEPTABLE);
+    }
+    Integer userId = DataElementHubRestApplication.getCurrentUser().getId();
+    String timestamp = new Timestamp(System.currentTimeMillis())
+        .toString().replaceAll("[ \\.\\-\\:]", "_");
+    exportService
+        .exportService(exportRequest, userId, format, fullExport, timestamp, exportDirectory);
+    return new ResponseEntity<>(timestamp, HttpStatus.ACCEPTED);
```

Comment:
The POST method creates a new resource here. The status code should be [201 CREATED](https://datatracker.ietf.org/doc/html/rfc7231#section-6.3.2) - supplying the link to the newly created export resource via [Location Header](https://datatracker.ietf.org/doc/html/rfc7231#section-7.1.2)

Reply:
The POST method accepts the request and starts an asynchronous export process, but the processing has not been completed. (According to [RFC](https://datatracker.ietf.org/doc/html/rfc7231#section-6.3.3) 202 ACCEPTED is the right status code)

Reply:
Ok so if the export resource is not necessarily available at the point the reply is sent, you are of course correct. I assumed it would be created - but be in a "pending" (or whatever) state, in which case 201 would be right.

(The origin server MUST create the resource before returning the 201 status code. If the action cannot be carried out immediately, the server SHOULD respond with 202 (Accepted) response instead.)